README.md

```diff
@@ -94,6 +94,39 @@ Available query commands:
   - Options:
     - `--rtl`: Display fields in right-to-left order
 
+### Evaluating Query Performance
+
+The `botnim evaluate` command allows you to evaluate the performance of queries against a vector store:
+
+```bash
+# Basic usage
+botnim evaluate takanon legal_text staging path/to/query_evaluations.csv
+
+# With custom parameters
+botnim evaluate takanon legal_text staging path/to/query_evaluations.csv --max-results 30 --adjusted-f1-limit 10
+```
+
+Required CSV columns:
+- `question_id`: Unique identifier for each question
```

Comment:
Does this mean that for a single question we'd have multiple rows in the evaluation csv? Might be a bit of a maintenance hassle (having to copy all details of a question many times).
I think that a structured yaml format might be more comfortable for editing - wdyt?

Reply:
@akariv 

> Does this mean that for a single question we'd have multiple rows in the evaluation csv?

yes.

> I think that a structured yaml format might be more comfortable for editing - wdyt?

let's discuss. the problem I see with an output yaml is what can we do with it as a next step (how to explore the results)  - the csv format enables a quick filtering and sorting. I might be missing something.